src/favorite-chips/index.tsx

```diff
@@ -108,7 +108,9 @@ const FavoriteChips: React.FC<Props> = ({
             text={t(FavoriteTexts.chips.addFavorite)}
             accessibilityRole="button"
             icon={Add}
-            onPress={() => navigation.navigate('AddEditFavorite', {})}
+            onPress={() =>
+              navigation.navigate('AddEditFavorite', {screen: 'SearchLocation'})
```

Comment:
Educate me, what makes the 'screen' property mandatory when navigating to AddEditFavorite?

Reply:
It's a sub screen. So two nested navigation stacks. `AddEditFavorite` is the root screen which is a stack itself and `SearchLocation` is a screen within that stack.

Reply:
What threw me off is that it was a mandatory parameter, but we have several other sub screens in the app which is navigated to without specifying the screen parameter. But I think I found it now. If I understand correctly it is the ´NavigatorScreenParams` which wraps the route param type that enables, and makes mandatory, the `screen` property.